interface/modules/custom_modules/oe-module-prior-authorizations/vendor/openemr/oe-module-installer-plugin/.gitignore

```diff
@@ -0,0 +1,3 @@
+vendor
```

Comment:
does changing this to `./vendor/*` remove a bunch of unnecessary files for this PR

Reply:
Let's talk about the way I built the module. I used composer to have the module not depend on the OpenEMR namespace. So, the module has its own namespace and vendor folder that the composer created. The vendor folder contains the autoloader for the module only. So, the module does not pollute the OpenEMR namespace.  The vendor folder is essential to the module to my knowledge. It is not full of unnecessary files.